DESIGN.md

```diff
@@ -18,21 +19,50 @@ Feel free to raise issues/file pull requests to this Github repository to help u
   - [Invocation](#invocation)
   - [Code Best Practices](#code-best-practices)
 
+# Components
+
+The vCenter Event Broker Appliance follows a highly modular approach, using Kubernetes and containers as an abstraction layer between the base operating system ([Photon OS](https://github.com/vmware/photon)) and the required application services. Currently the following components are used in the appliance:
+
+- VMware Event Router ([Github](https://github.com/vmware-samples/vcenter-event-broker-appliance/vmware-event-router))
+  - Supported Event Stream Sources:
+    - VMware vCenter ([Website](https://www.vmware.com/products/vcenter-server.html))
+  - Supported Event Stream Processors: 
+    - OpenFaaS ([Website](https://www.openfaas.com/))
+    - AWS EventBridge ([Website](https://aws.amazon.com/eventbridge/))
+- Contour ([Github](https://github.com/projectcontour/contour))
+- Kubernetes ([Github](https://github.com/kubernetes/kubernetes))
+- Photon OS ([Github](https://github.com/vmware/photon))
+
+
+The VMware Event Router implements the core functionality of VEBA, that is connecting to event `streams` ("sources") and processing the events with a configurable event `processor` such as OpenFaaS or AWS EventBridge.
```

Comment:
I think we said we'd fully spell out "VEBA" to be vCenter Event Broker Appliance. This looks to be the only place in the text that uses the acronym.

Reply:
Good catch.